### PR TITLE
feat(payments): INT-3610 added Mollie Checkout

### PR DESCRIPTION
**Pull request by JoseAlvarezHdz**
## What? [INT-3610](https://jira.bigcommerce.com/browse/INT-3610)
Create Custom Card for Mollie payment method on checkout and PaymentMethod

## Why?
In order to be able to purchase through Mollie

## Testing / Proof

[Video](https://drive.google.com/file/d/1ZYGejKPOXk_jhI99w0pdcIA_vIFd7FwN/view?usp=sharing)

screen-recorder-wed-dec-30-2020-11-32-54 (1)

## Dependencies

[SDK](https://github.com/bigcommerce/checkout-sdk-js/pull/1052)

@bigcommerce/checkout

### CODE REVIEW

File: src/app/payment/paymentMethod/PaymentMethodId.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ enum PaymentMethodId {
     StripeGooglePay = 'googlepaystripe',
     StripeV3 = 'stripev3',
     Zip = 'zip',
+    Mollie = 'mollie',
```
Comment (vmparra) [inline src/app/payment/paymentMethod/PaymentMethodId.ts:36]: Can you please add this PaymentMethodId alphabetically?

**Review by vmparra — CHANGES_REQUESTED** [review]
Thanks @JoseAlvarezHdz for doing this, I just left you a few comments. Please let me know if you have questions.

File: src/app/payment/paymentMethod/PaymentMethodId.ts (hunk 1 above)
Comment (JoseAlvarezHdz) [reply src/app/payment/paymentMethod/PaymentMethodId.ts]: its done

**Comment by JoseAlvarezHdz** [comment]
@vmparra i did the changes you mentions, can you validate them please?

**Review by vmparra — APPROVED** [review]
LGTM 👍🏻

**Review by capsula4 — APPROVED** [review]
Great job! 👍

**Comment by PascalZajac** [comment]
@JoseAlvarezHdz I think this may need rebasing in order to make the build pass?

**Comment by JoseAlvarezHdz** [comment]
@PascalZajac done!

**Pull request merged**